http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeoAuthProvider.scala

```diff
@@ -46,10 +46,9 @@ object SamResource {
   class AppSamResource extends SamResource[AppSamResourceId] {
     def resourceIdAsString(r: AppSamResourceId): String = r.resourceId
     def resourceType(r: AppSamResourceId): SamResourceType = r.resourceType
-    def policyNames(r: AppSamResourceId): Set[SamPolicyName] = r.resourceType match {
-      case SamResourceType.SharedApp => Set(SamPolicyName.Owner, SamPolicyName.User)
-      case _                         => Set(SamPolicyName.Creator, SamPolicyName.Manager)
-    }
+    // Union of shared and private kubernetes-app roles
```

Comment:
Why this change? Private app should not have the same set of policy names as shared apps no?

Reply:
Yeah this kind of relates to my above Sam comment -- basically when we get the list of `AppSamResourceId`s from Sam, we lose the `accessScope` field. So just testing all the policy names when filtering down the Sam result (this is only used for listing resources).

… As I'm thinking about this more though, I wonder if it would be better to actually fix the Sam deserialization to _not_ lose `accessScope`. Let me take another pass at doing this cleaner…